munch-match/src/App.css

```diff
@@ -144,3 +144,13 @@ h5 {
     transform: rotate(360deg);
   }
 }
+
+.map {
+  width: 50%;
+  height: 400px;
```

Comment:
Just for future reference, try to avoid using pixels for dimensions; using % is usually better (from my knowledge) as this scales and allows for more responsiveness! However, for our current situation what you have done is great. :)